raitools/raitools/rai_analyzer/constants.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+"""Defines common constants."""
+
+from enum import Enum
+
+
+class ModelTask(str, Enum):
+    """Provide model task constants. Can be 'classification' or 'regression'.
+    """
+
+    CLASSIFICATION = 'classification'
+    REGRESSION = 'regression'
+
+
+class ManagerNames(str):
+    """Provide the manager names."""
+
+    CAUSAL = 'causal'
+    COUNTERFACTUAL = 'counterfactual'
+    ERROR_ANALYSIS = 'error_analysis'
+    EXPLAINER = 'explainer'
+    FAIRNESS = 'fairness'
+
+
+class Metadata(str):
+    """Provide constants for metadata and saved files."""
+
+    META_JSON = 'meta.json'
+    MODEL = 'model'
+
+
+class ListProperties(str):
+    """Provide constants for listing manager properties."""
+
+    MANAGER_TYPE = 'manager_type'
+
+
+class ExplainerManagerKeys(str):
```

Comment:
Could ExplainerManagerKeys be private? If this is only meant for internal consumption and doesn't serve any purpose to the end user it should be private.

Reply:
done!